Sources/System/Internals/Constants.swift

```diff
@@ -0,0 +1,471 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+// For platform constants redefined in Swift. We define them here so that
+// they can be used anywhere without imports and without confusion to
+// unavailable local decls.
+
+import CSystem
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#elseif os(Linux) || os(FreeBSD) || os(Android)
+import Glibc
+#elseif os(Windows)
+import ucrt
+#else
+#error("Unsupported Platform")
+#endif
+
+// MARK: errno
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+@_alwaysEmitIntoClient
+internal var _ERRNO_NOT_USED: CInt { 0 }
+#endif
+
+@_alwaysEmitIntoClient
+internal var _EPERM: CInt { EPERM }
```

Comment:
I think we should just get rid of these. Is there a good reason to keep them?

Reply:
They help avoid the `#if/#elseif/#else` import dance everywhere, and (I didn't check the actual impact of this) they might help when there are other `@unavailable(renamed:)` members with the same name nearby.

Reply:
Don't we need that dance in some files anyway, to call the functions or use types not exposed in CInterop? 

I imagine maintaining this would be more work than copy and pasting the right import incantation.

Reply:
Oh, unavailable names shadowing these is a great point! We can't easily resolve it by qualifying with the module name, because that's different between Darwin and Glibc. Bah.

If only we could say `import Darwin as _System` or something

Reply:
We only do the import dance inside the `Internals` sub-folder (pending CTypes.swift in upcoming commit). I do agree that the dance alone is a toss up, but I really don't want to have to refer to explicit modules anywhere for constants.